File: 백준/Bronze/4153. 직각삼각형/직각삼각형.c
#include <stdio.h>

int main() {
    int A, B, C;

    while(1) {
        scanf("%d %d %d", &A, &B, &C);
        if (A == 0 && B == 0 && C == 0) {
            break;
        }
        if(A > B) {
            int temp = A;
            A = B;
            B = temp;
        } 
        if (B > C) {
            int temp = B;
            B = C; 
            C = temp;
        }

        if(A*A + B*B == C*C) {
            printf("right\n");
        } else {
            printf("wrong\n");
        }
    }

    return 0;
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while(True):
    A, B, C = map(int, input().split())
    if A == 0 and B == 0 and C == 0:
        break
    if A > B:
        A, B = B, A
    if B > C:
        B, C = C, B
    
    if A*A + B*B == C*C:
        print("right")
    else :
        print("wrong")